tokens/src/core/global/elevation.json

```diff
@@ -1,19 +1,19 @@
 {
   "elevation": {
     "zindex": {
-      "0": { "value": 0, "type": "ratio" },
-      "200": { "value": 200, "type": "ratio" },
-      "400": { "value": 400, "type": "ratio" },
-      "600": { "value": 600, "type": "ratio" },
-      "800": { "value": 800, "type": "ratio" },
-      "1000": { "value": 1000, "type": "ratio" },
-      "1200": { "value": 1200, "type": "ratio" },
-      "1400": { "value": 1400, "type": "ratio" },
-      "1600": { "value": 1600, "type": "ratio" },
-      "1800": { "value": 1800, "type": "ratio" },
-      "2000": { "value": 2000, "type": "ratio" },
-      "sticky": { "value": 1020, "source": "$zindex-sticky", "type": "ratio" },
-      "fixed": { "value": 1030, "source": "$zindex-fixed", "type": "ratio" }
+      "0": { "value": 0, "type": "ratio", "description": "z-index of level 0." },
+      "200": { "value": 200, "type": "ratio", "description": "z-index of level 200." },
+      "400": { "value": 400, "type": "ratio", "description": "z-index of level 400." },
+      "600": { "value": 600, "type": "ratio", "description": "z-index of level 600." },
+      "800": { "value": 800, "type": "ratio", "description": "z-index of level 800." },
+      "1000": { "value": 1000, "type": "ratio", "description": "z-index of level 1000." },
+      "1200": { "value": 1200, "type": "ratio", "description": "z-index of level 1200." },
+      "1400": { "value": 1400, "type": "ratio", "description": "z-index of level 1400." },
+      "1600": { "value": 1600, "type": "ratio", "description": "z-index of level 1600." },
+      "1800": { "value": 1800, "type": "ratio", "description": "z-index of level 1800." },
+      "2000": { "value": 2000, "type": "ratio", "description": "z-index of level 2000." },
```

Comment:
These feel like another example of the "do we expect to only see the descriptions?" question.

Reply:
Yeah, that's a good example of redundant description if we are able to see variable name, which we most likely will.
I can't come up with any meaningful description here, maybe we should just leave it empty.

@adamstankiewicz curious what you think

Reply:
Let's leave as is for the time being, but plan on revisiting once we have more clarity how these descriptions will ultimately get used. Goal of this PR is largely to get some reasonable descriptions in place to solidify the decision to do so and showcase a few examples.